libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -59,4 +61,12 @@ export class BookSearchComponent implements OnInit {
       this.store.dispatch(clearSearch());
     }
   }
+
+  throttleSearchQuery(event) {
+    from(event)
+    .pipe(throttleTime(500))
+    .subscribe( () => {
```

Comment:
What happens when this control is destroyed? Isn't this a memory leak?

Reply:
updated